test/extended/threads/atomic.wast

```diff
@@ -4,6 +4,7 @@
   (memory 1 1 shared)
 
   (func (export "init") (param $value i64) (i64.store (i32.const 0) (local.get $value)))
+  (func (export "initOffset") (param $value i64) (param $offset i32) (i64.store (local.get $offset) (local.get $value)))
```

Comment:
I think these files comes from the standard. We should not add custom modifications here. They should go separate files.